website/pages/en/index.js

```diff
@@ -297,6 +269,20 @@ const Hero = ({ language }) => (
       </div>
 
       <MiniRepl language={language} />
+
+      <h3>Special Sponsor</h3>
+      <div class="sponsors-tier">
+        <a href="https://www.handshake.org" title="Handshake" target="_blank">
+          <img
+            src="https://handshake.org/images/landing/logo-dark.svg"
```

Comment:
I slightly prefer `https://handshake.org/images/landing/logo-light.svg`

Reply:
didn't think of that!